server/src/main/java/org/spine3/server/event/EventEntity.java

```diff
@@ -30,9 +32,21 @@
 /**
  * Stores an event.
  *
+ * <p>{@code EventEntity} is a Spine internal {@link org.spine3.server.entity.Entity Entity} type,
+ * which is not designed for the direct usage. Both API and behaviour of this class may are subjects
+ * of change.
+ *
  * @author Alexander Yevsyukov
+ * @author Dmytro Dashenkov
  */
-class EventEntity extends AbstractEntity<EventId, Event> {
+@Internal
+public class EventEntity extends AbstractEntity<EventId, Event> {
+
+    /**
+     * The name of the Entity Column representing the time, when the event was fired.
+     * @see #getTime()
+     */
+    static final String TIME_COLUMN = "time";
```

Comment:
Let's rename the property accordingly (see the related comment above). It's either a "created", or  "last modified".